docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1486,6 +1486,42 @@ FROM example.web.page_views
 WHERE "$file_modified_time" = CAST('2022-07-01 01:02:03.456 UTC' AS TIMESTAMP WITH TIME ZONE)
 ```
 
+(iceberg-system-tables)=
+#### System tables
+
+The connector exposes metadata tables in the system schema.
+
+##### `iceberg_tables` table
+
+The `iceberg_tables` table allows listing only iceberg tables from a given catalog.
+The `SHOW TABLES` statement, `information_schema.tables`, and `jdbc.tables` will all
+return all tables that exist in the underlying metastore, even if the table cannot
+be handled in any way by the iceberg connector. This can happen if other connectors
+like hive or delta, use the same metastore, catalog, and schema to store its tables.
+
+The table returns following columns:
+
+- `table_schema`
+  : The name of the schema the table is in.
+
+- `table_name`
+  : The name of the table.
```

Comment:
The style in the generated pages is a little dirty. We could change it to the table, or remove new lines. 
![Screenshot 2025-03-18 at 15 54 20](https://github.com/user-attachments/assets/94bc360c-fe87-4213-9e1d-d09d281ec92f)

Reply:
I used `docs/build` command. We shouldn't trust GitHub rendering.

Reply:
Ok, thanks. I changed it to the table and it looks like that now:
![image](https://github.com/user-attachments/assets/029bb4d8-0af9-4a56-a244-8de81eb219f0)